xsuite/src/world/fsworld.test.ts

```diff
@@ -105,6 +105,23 @@ test.concurrent("FSWorld.proxy.getAccount on empty U8A address", async () => {
   assertAccount(await world.proxy.getAccount(zeroU8AAddress), emptyAccount);
 });
 
+test.concurrent("FSWorld.proxy.blockNonce", async () => {
+  using world = await FSWorld.start();
+  const wallet = await world.createWallet({
+    balance: 10n ** 18n,
+  });
+  await wallet.setAccount({
+    balance: 2n * 10n ** 18n,
+  });
+  assertAccount(await wallet.getAccount(), {
```

Comment:
For fetching the data, let's use a proxy directly, and let's also test the `blockNonce` at the initialization of the proxy.

Reply:
Do you mean creating another proxy? Or accessing the world's proxy explicitly?

What do you mean by "test the `blockNonce`"?

Reply:
We can use the world to set the states. And then create another instance of the proxy to do the data fetching.

And we don't test setting blockNonce in the constructor, i.e. `new Prxoy({..., blockNonce: ...})`.